<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
  "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
<head>
<meta name="generator" content="jemdoc, see http://jemdoc.jaboc.net/" />
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<link rel="stylesheet" href="jemdoc.css" type="text/css" />
<title>Li Ma - Research</title>
</head>
<body>
<table summary="Table for page layout." id="tlayout">
<tr valign="top">
<td id="layout-menu">
<div class="menu-item"><a href="index.html">Home</a></div>
<div class="menu-item"><a href="People.html">People</a></div>
<div class="menu-item"><a href="Bio.html">Bio</a></div>
<div class="menu-item"><a href="Research.html" class="current">Research</a></div>
<div class="menu-item"><a href="Teaching.html">Teaching</a></div>
<div class="menu-item"><a href="cv.pdf">CV</a></div>
</td>
<td id="layout-content">
<div id="toptitle">
<h1>Li Ma</h1>
</div>
<h3>Research interests</h3>
<ul>
<li><p>Generative models</p>
</li>
<li><p>Recursive partitioning and tree-related methods</p>
</li>
<li><p>Nonparametric modeling and inference</p>
</li>
<li><p>Multi-scale inference</p>
</li>
<li><p>Statistical modeling of biomedical data sets, especially microbiome sequencing data and flow cytometry<br /></p>
</li>
</ul>
<p>A recent methodological focus of my research is on using multi-scale techniques to construct flexible probability models that can be applied to massive data sets. Traditional nonparametric approaches, while enjoying many established theoretical properties, are often computationally intractable for big data. Multi-scale inference provides a general framework for tackling the computational bottleneck, while preserving the theoretical guarantees enjoyed by classical methods.</p>
<p>My applied interest focuses on modeling complex data sets from biomedical experiments. In particular, current efforts have been devoted to modeling and analyzing data from microbiome sequencing experiments and flow cytometry.</p>
<h3>Support</h3>
<p>My research group is currently supported by both the NSF (DMS-2152999 and EEC-2133504) and the NIH (NIGMS grant R01-GM135440). Prior support: NSF (DMS-1309057, DMS-1612889, DMS-1749789, and DMS-2013930) and a <a href="http://research.google.com/research-outreach.html#/research-outreach/faculty-engagement/faculty-research-awards">Google Faculty Research Award</a>.</p>
<h3>Preprints</h3>
<p>Awaya N, Xu Y, and Ma L. (2025) Two-sample comparison through additive tree models for density ratios. <a href="https://arxiv.org/abs/2508.03059">[arxiv]</a><a href="https://youtu.be/-8pFpWZCo-k?si=wZ3rf0CMhUbKNi4m">[plenary talk@bnp14]</a></p>
<p>Liu L and Ma L. (2025) On forest-type tree ensemble approaches to density learning under a
generalized Bayesian framework.</p>
<p>Ma L and Bruni B. (2025) A partial likelihood approach to tree-based density modeling and its application in Bayesian inference. <a href="https://arxiv.org/abs/2412.11692">[arxiv]</a><a href="https://drive.google.com/file/d/17xgNihmN2TnrYUZ3-OiCk-N7QJhfk_KS/view?usp=drive_link">[talk]</a></p>
<p>Wang Z, Awaya N, and Ma L. (2024) Generative modeling of conditional densities through additive tree flows. <a href="https://arxiv.org/abs/2406.05260">[arxiv]</a></p>
<p>Wang Z, Mao J, and Ma L. (2024) A tree-based model for addressing sparsity and taxa covariance in microbiome compositional count data. <a href="https://www.imsi.institute/videos/microbiome-analyses-through-tree-based-models-for-high-dimensional-compositions/">[talk]</a><a href="https://arxiv.org/abs/2106.15051">[arxiv]</a>
</p>
<p>Kim H, Siddiqui N, Karstens L, and Ma L. (2024) A negative binomial latent factor model for paired microbiome sequencing data. <a href="https://doi.org/10.1101/2024.12.01.626246">[bioRxiv]</a></p>
<p>Luo H, Horiguchi A, and Ma L. (2024) Efficient decision trees for tensor regressions. <a href="https://arxiv.org/abs/2408.01926">[arxiv]</a><a href="https://github.com/hrluo/TensorDecisionTreeRegressor">[code]</a></p>
<p>Horiguchi A, Ma L, and Szabo B. (2024) Sampling depth trade-off in function estimation under a two-level design. <a href="http://arxiv.org/abs/2310.02968">[arxiv]</a></p>
<h3>Publications</h3>
<p>Wei G and Ma L. (2025) Stream-level flow matching with Gaussian processes. ICML 2025. <a href="https://openreview.net/pdf?id=qg9p1I5lmp">[online]</a></p>
<p>Horiguchi A, Chan C, and Ma L. (2024) A tree perspective on stick-breaking models in covariate-dependent mixtures (with discussion). <i>Bayesian Analysis.</i> (To appear). <a href="https://doi.org/10.1214/24-BA1462">[online]</a><a href="https://arxiv.org/abs/2208.02806">[arxiv]</a></p>
<p>Awaya N and Ma L. (2024) Unsupervised tree boosting for learning probability distributions. <i>Journal of Machine Learning Research.</i> 25(198):1−52. <a href="https://www.jmlr.org/papers/v25/22-0980.html">[online]</a><a href="https://github.com/MaStatLab/boostPM">[R package]</a></p>
<p>Liu L and Ma L. (2024) Spatial adaptation by Bayesian unsupervised trees. <i>Proceedings of Thirty Seventh Conference on Learning Theory, PMLR.</i> 247:3556-3581. <a href="https://proceedings.mlr.press/v247/liu24b.html">[online]</a></p>
<p>Liu R, Li M, and Ma L. (2024) Efficient in-situ image and video compression through probabilistic image representation. <i>Signal Processing.</i> Vol. 215, 109268. <a href="https://doi.org/10.1016/j.sigpro.2023.109268">[online]</a><a href="https://arxiv.org/abs/1912.05622">[arxiv]</a><a href="https://github.com/xylimeng/CARP">[Matlab code]</a> </p>
<p>Gorsky S, Chan C, and Ma L. (2024) Coarsened mixtures of hierarchical skew normal kernels for flow cytometry analyses. <i>Bayesian Analysis.</i> Vol.19, No.2, 439 - 463. <a href="https://vimeo.com/571655586">[<b>15-min video</b>]</a><a href="https://doi.org/10.1214/22-BA1356">[online]</a><a href="https://cran.r-project.org/web/packages/COMIX/index.html">[R package]</a></p>
<p>Awaya N and Ma L. (2024) Hidden Markov Pólya trees for high-dimensional distributions. <i>Journal of the American Statistical Association, Theory and Methods.</i>  Vol.119, No.545, 189–201. <a href="https://doi.org/10.1080/01621459.2022.2105223">[online]</a><a href="https://arxiv.org/abs/2011.03121">[arxiv]</a><a href="https://github.com/MaStatLab/SMCMP">[R package]</a><a href="Research/talks/slss_pt.pdf">[slides]</a></p>
<p>Ji Z and Ma L. (2023) Controlling taxa abundance improves metatranscriptomics differential analysis. <i>BMC Microbiology. 24:60.</i> <a href="http://dx.doi.org/10.1186/s12866-023-02799-9">[online]</a></p>
<p>LeBlanc P and Ma L. (2023) Microbiome subcommunity learning with logistic-tree normal latent Dirichlet allocation. <i>Biometrics.</i> Vol. 79, Iss. 3, 2321-2332. <a href="https://doi.org/10.1111/biom.13772">[online]</a><a href="https://arxiv.org/abs/2109.05386">[arxiv]</a><a href="https://github.com/PatrickLeBlanc/LTNLDA">[R package]</a></p>
<p>Gorsky S and Ma L. (2022) Multiscale Fisher's independence test for multivariate dependence (with discussion). <i>Biometrika.</i> Vol. 109, No. 3, 569–587. <a href="https://doi.org/10.1093/biomet/asac013">[online]</a><a href="https://arxiv.org/abs/1806.06777">[arxiv]</a> <a href="https://cran.r-project.org/web/packages/MultiFit/index.html">[R package]</a></p>
<p>Gorsky S and Ma L. (2022) Rejoinder: &ldquo;Multiscale Fisher's independence test for multivariate dependence.&rdquo; <i>Biometrika.</i> Vol. 109, No. 3, 605-609. <a href="https://doi.org/10.1093/biomet/asac034">[online]</a></p>
<p>Mao J and Ma L. (2022) Dirichlet-tree multinomial mixtures for clustering microbiome compositions. <i>Annals of Applied Statistics.</i> Vol. 16, No. 3, 1476-1499. <a href="https://www.imsi.institute/videos/microbiome-analyses-through-tree-based-models-for-high-dimensional-compositions/">[<b>talk</b>]</a><a href="https://vimeo.com/570047013">[<b>15-min video</b>]</a><a href="https://doi.org/10.1214/21-AOAS1552">[online]</a><a href="https://arxiv.org/abs/2008.00400">[arxiv]</a><a href="https://github.com/MaStatLab/DTMM">[R package]</a><a href="https://github.com/MaStatLab/DTMM_reproducible">[numerical examples]</a></p>
<p>Siddiqui N, Ma L, Brubaker L, Mao J, Hoffman C, Wang Z, Karstens L. (2022) Updating urinary microbiome analyses to enhance biologic interpretation. <i>Frontiers in Cellular and Infection Microbiology. 12:789439. </i> <a href="https://doi.org/10.3389/fcimb.2022.789439">[online]</a></p>
<p>Vaughan M, Zemtsov G.E., Dahl E.M., Karstens L, Ma L, Siddiqui N. (2022) Concordance of urinary microbiota detected by 16S rRNA amplicon sequencing versus expanded quantitative urine culture. <i>American Journal of Obstetrics & Gynecology.</i> <a href="https://doi.org/10.1016/j.ajog.2022.06.031">[online]</a></p>
<p>Luo K, Zhong J, Safi A, Hong L.K., Tewari A.K., Song L, Reddy T.E., Ma L, Crawford G.E., and Hartemink A.J. (2022) 
Quantitative occupancy of myriad transcription factors from one DNase experiment enables efficient comparisons across conditions. <i>Genome Research.</i> 32: 1183-1198. <a href="https://genome.cshlp.org/content/32/6/1183">[online]</a><a href="https://doi.org/10.1101/2020.06.28.171587">[bioRxiv]</a></p>
<p>Li M and Ma L. (2022) Learning asymmetric and local features in multi-dimensional data through wavelets with recursive partitioning. <i>IEEE Transactions on Pattern Analysis and Machine Intelligence.</i> Vol. 44, No. 11, 7674-7687. <a href="https://doi.org/10.1109/TPAMI.2021.3110403">[online]</a><a href="https://arxiv.org/abs/1711.00789">[arxiv]</a><a href="https://github.com/xylimeng/WARP">[Matlab toolbox & R package]</a></p>
<p>Vaughan M, Mao J, Karstens L, Ma L, Amundsen C, Schmader K, Siddiqui N. (2021). The urinary microbiome in postmenopausal women with recurrent urinary tract infections. <i>Journal of Urology.</i> Vol. 206, No. 5, 1222-1231. <a href="https://www.auajournals.org/doi/abs/10.1097/JU.0000000000001940">[online]</a><a href="https://doi.org/10.1101/2020.12.21.423901">[bioRxiv]</a></p>
<p>Ramalingam S, Siamakpour-Reihani S, Bohannan L, Ren Y, Sibley, Sheng J, Ma L, Nixon AB, Lyu J, Parker DC, Bain B, Muehlbauer M, Ilkayeva O, Kraus VB, Huebner J, Spitzer T, Brown J, Peled J, van den Brink M, Gomes A, Choi T, Gasparetto C, Horwitz M, Long G, Lopez R, Rizzieri D, Sarantopoulos S, Chao N, and Sung AD. (2021). A phase 2 trial of the somatostatin analog pasireotide to prevent GI toxicity and acute GVHD in allogeneic hematopoietic stem cell transplant. <i>PLOS ONE.</i> Vol. 16, No. 6. <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0252995">[online]</a>.</p>
<p>Giri V, Kegerreis K, Ren Y, Bohannon L, Lobaugh-Jin E, Messina J, Matthews A, Mowery Y, Sito E, Lassiter M, Saullo J, Jung S, Ma L, Greenberg M, Andermann T, van den Brink M, Peled J, Gomes A, Choi T, Gasparetto C, Horwitz M, Long G, Lopez R, Rizzieri D, Sarantopoulos S, Chao N, Allen D, and Sung A. (2021). Chlorhexidine gluconate bathing reduces the incidence of bloodstream infections in adults undergoing inpatient hematopoietic cell transplantation. <i>Transplantation and Cellular Therapy.</i> Vol. 27, No. 1, 262e1-e11. <a href="https://doi.org/10.1016/j.jtct.2021.01.004">[online]</a></p>
<p>Liu R, Li M, and Ma L. (2020) CARP: Compression through adaptive recursive partitioning for multi-dimensional images. <i>CVPR 2020: IEEE/CVF Conference on Computer Vision and Pattern Recognition.</i> <a href="https://openaccess.thecvf.com/content_CVPR_2020/papers/Liu_CARP_Compression_Through_Adaptive_Recursive_Partitioning_for_Multi-Dimensional_Images_CVPR_2020_paper.pdf">[online]</a><a href="https://github.com/xylimeng/CARP">[Matlab code]</a></p>
<p>Christensen J and Ma L. (2020) A Bayesian hierarchical model for related densities using Pólya trees. <i>
Journal of the Royal Statistical Society. Series B.</i>, Vol. 82, 127-153. <a href="https://doi.org/10.1111/rssb.12346">[online]</a> <a href="https://arxiv.org/abs/1710.01702">[preprint]</a> <a href="https://github.com/MaStatLab/HAPT-package">[R package]</a></p>
<p>Mao J, Chen Y, and Ma L. (2020) Bayesian graphical compositional regression for microbiome data. <i>Journal of the American Statistical Association, Applications and Case Studies.</i> Vol. 115, No. 530, 610-624. <a href="https://www.imsi.institute/videos/microbiome-analyses-through-tree-based-models-for-high-dimensional-compositions/">[<b>talk</b>]</a><a href="https://doi.org/10.1080/01621459.2019.1647212">[online]</a><a href="https://arxiv.org/abs/1712.04723">[preprint]</a><a href="https://github.com/MaStatLab/BGCR">[R package]</a><a href="https://github.com/MaStatLab/BGCR_analysis">[Source code for examples]</a></p>
<p>Ma L. (2019) Discussion on &ldquo;Latent Nested Nonparametric Priors&rdquo; by Camerlanghi et al. <i>Bayesian Analysis.</i> Vol. 14, No. 4, 1303-1356. <a href="https://www.doi.org/10.1214/19-BA1169">[online]</a><a href="files/discussion_on_LNNP.pdf">[preprint]</a></p>
<p>Ma L and Mao J. (2019) Fisher exact scanning for dependency. <i>Journal of the American Statistical Association, Theory and Methods.</i> Vol. 114, No. 525, 245-258. <a href="https://doi.org/10.1080/01621459.2017.1397522">[online]</a><a href="http://arxiv.org/abs/1608.07885">[preprint]</a><a href="https://github.com/MaStatLab/FES">[R code]</a></p>
<p>Soriano J and Ma L. (2019) Mixture modeling on related samples through psi-stick breaking and kernel perturbation. <i>Bayesian Analysis</i>. Vol. 14, No.1, 161-180. <a href="https://projecteuclid.org/euclid.ba/1524124868">[online]</a><a href="https://github.com/jacsor/cremid">[R package]</a><a href="https://github.com/jacsor/MPG-examples">[examples]</a></p>
<p>Ma L and Soriano J. (2018) Analysis of distributional variation through multi-scale Beta-Binomial modeling. <i>Journal of Computational and Graphical Statistics</i>. Vol. 27, No. 3, 529-541. <a href="https://doi.org/10.1080/10618600.2017.1402774">[online]</a><a href="http://arxiv.org/abs/1604.01443">[preprint]</a><a href="https://cran.r-project.org/web/packages/MRS/index.html">[R package]</a></p>
<p>Tang Y, Ma L, and Nicolae DL. (2018) A phylogenetic scan test on Dirichlet-tree multinomial model for microbiome data. <i>Annals of Applied Statistics</i>. Vol. 12, No. 1, 1-26. <a href="https://projecteuclid.org/euclid.aoas/1520564463">[online]</a><a href="https://arxiv.org/pdf/1610.08974.pdf">[preprint]</a><a href="https://github.com/yunfantang/phyloDM">[R code]</a></p>
<p>Ma L and Soriano J. (2018) Efficient functional ANOVA through wavelet-domain Markov groves. <i>Journal of the American Statistical Association, Theory and Methods.</i> Vol. 113, No. 3, 802-818. <a href="http://amstat.tandfonline.com/doi/full/10.1080/01621459.2017.1286241?src=recsys">[online]</a><a href="https://cran.r-project.org/web/packages/grove/index.html">[R package]</a></p>
<p>Soriano J and Ma L. (2017) Probabilistic multi-resolution scanning for two-sample differences. <i>Journal of the Royal Statistical Society. Series B.</i> Vol. 79, No. 2, 547-572. <a href="http://onlinelibrary.wiley.com/doi/10.1111/rssb.12180/abstract">[online]</a><a href="https://cran.r-project.org/web/packages/MRS/index.html">[R package]</a></p>
<p>Ma L. (2017) Recursive partitioning and multi-scale modeling on conditional densities. <i>Electronic Journal of Statistics</i>. Vol. 11, No. 1, 1297-1325. <a href="http://projecteuclid.org/euclid.ejs/1492135235">[online]</a><a href="https://github.com/MaStatLab/PTT">[R package]</a></p>
<p>Ma L. (2017) Adaptive shrinkage in Pólya tree type models. <i>Bayesian Analysis.</i> Vol. 12, No. 3, 779-805. (<i>Featured in the editor's invited session &ldquo;Highlights from Bayesian Analysis&rdquo; at JSM 2017.)</i>
<a href="https://projecteuclid.org/euclid.ba/1473276260">[online]</a><a href="files/APT_supp.pdf">[supplement]</a><a href="https://github.com/MaStatLab/PTT">[R package]</a></p>
<p>Ma L. (2015) Scalable Bayesian model averaging through local information propagation. <i>Journal of the American Statistical Association, Theory and Methods</i>. Vol. 110, No. 510, 795-809. <a href="http://www.tandfonline.com/doi/full/10.1080/01621459.2014.980908">[online]</a><a href="http://arxiv.org/abs/1403.2397">[preprint]</a><a href="https://github.com/MaStatLab/LIPS">[R package]</a></p>
<p>Ma L. (2013) Adaptive testing of conditional association through recursive mixture modeling. <i>Journal of the American Statistical Association, Theory and Methods</i>. Vol. 108, No. 504, 1493-1505. <a href="http://www.tandfonline.com/doi/full/10.1080/01621459.2013.838899">[online]</a><a href="https://github.com/MaStatLab/remcat">[R package]</a></p>
<p>Ma L, Wong WH, and Owen AB. (2012) A sparse transmission disequilibrium test for haplotypes based on Bradley-Terry graphs. <i>Human Heredity</i>. Vol. 73, No. 1, 52-61. <a href="http://content.karger.com/produktedb/produkte.asp?DOI=10.1159/000335937">[online]</a><a href="files/sparseTDT.pdf">[preprint]</a></p>
<p>Ma L and Wong WH. (2011) Coupling optional Pólya trees and the two sample problem. 
<i>Journal of the American Statistical Association, Theory and Methods</i>. Vol. 106, No. 496, 1553-1565. <a href="http://www.tandfonline.com/doi/abs/10.1198/jasa.2011.tm10003">[online]</a><a href="http://arxiv.org/pdf/1011.1253.pdf">[arxiv]</a></p>
<p>Ma L, Stein ML, Wang M, Shelton AO, Pfister CA, and Wilder KJ. (2011) 
A method for unbiased estimation of population abundance along curvy
margins. <i>Environmetrics</i>. Vol. 22, No. 3, 330-339. <a href="http://onlinelibrary.wiley.com/doi/10.1002/env.1053/abstract">[online]</a></p>
<p>Wong WH and Ma L. (2010) Optional Pólya tree and Bayesian inference. 
<i>Annals of Statistics</i>. Vol. 38, No. 3, 1433-1459. 
<a href="https://doi.org/10.1214/09-AOS755">[online]</a><a href="files/optree.pdf">[pdf]</a><a href="https://github.com/MaStatLab/PTT">[R package]</a></p>
<p>Ma L, Assimes T, Asadi NB, Iribarren C, Quertermous T, and Wong WH. (2010) 
An &ldquo;almost exhaustive&rdquo; search based sequential permutation method for detecting epistasis 
in disease association studies. <i>Genetic Epidemiology</i>. Vol. 34, No. 5, 434-443. 
<a href="http://www.ncbi.nlm.nih.gov/pubmed/20583286">[online]</a><a href="Software/sFPgrowth.html">[software]</a></p>
<p>Ma L, Mease D, and Russell D. (2010) A four group cross-over design for
measuring irreversible treatments on web search
tasks. <i>Proceedings of HICSS-44</i>. <a href="http://doi.ieeecomputersociety.org/10.1109/HICSS.2011.11">[online]</a></p>
<div id="footer">
<div id="footer-text">
Page generated 2025-08-13 08:41:32 CDT, by <a href="http://jemdoc.jaboc.net/">jemdoc</a>.
</div>
</div>
</td>
</tr>
</table>
</body>
</html>
